### DISCUSSION TITLE
The math formula on the github web page does not display properly

### DISCUSSION THREAD

**Discussion by digger-yu** [community/community#52991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

hi all 
I found that some mathematical formulas are not displayed properly on the github web page, but they are normal in vscode

like 
```
$$
\left[\begin{matrix} X_{10} & X_{11} \\ X_{00} & X_{01} \end{matrix} \right]
\text{~and~}
\left[\begin{matrix} A_{10} & A_{11} \\ A_{00} & A_{01} \end{matrix} \right]。
$$
```

When you access it with chrome, it looks like this
![image](https://user-images.githubusercontent.com/55081697/232360304-575cb911-93ec-4836-a105-5081f6452a73.png)

But when you open it in vscode, it looks like this
![image](https://user-images.githubusercontent.com/55081697/232360511-dc9e4b3c-1615-4a68-84df-3a20fd28552c.png)

I think the way it is rendered on the web makes it difficult to understand the meaning of the mathematical formula itself, is this a problem?

**Comment by nschloe** [comment]
This is one of many bugs in GitHub's math renderer. I've added it to [the list](https://github.com/nschloe/github-math-bugs#matrices-without-line-breaks-in--math).

Workarounds:

- Use triple backtick math. (I always recommend this, many bugs are avoided here.)
- Insert a line break after `\\`.

**Reply by digger-yu** [reply]
> This is one of many bugs in GitHub's math renderer. I've added it to [the list](https://github.com/nschloe/github-math-bugs#matrices-without-line-breaks-in--math).
> 
> Workarounds:
> 
> * Use triple backtick math. (I always recommend this, many bugs are avoided here.)
> * Insert a line break after `\\`.

Thank you for your reply
Before I changed it, the ```math``` formulas were not displayed properly on the web, after using ```math```, although they are displayed properly on the web, they are not displayed in the vscode preview,
As shown
![image](https://user-images.githubusercontent.com/55081697/232410716-dc22cc78-5e3b-447c-b38a-0a5e10ef0af4.png)

 I haven't found a way to display both other than using images
